### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by santipalenque**

### CODE REVIEW

File: src/routes/PrivateRoute.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const PrivateRoute = ({
       setReFetchedUserProfile(true);
       getUserProfile();
     }
-  }, [reFetchedUserProfile, isLoggedIn, hasTicket, isAuthorized, userProfile]);
+  }, [reFetchedUserProfile, isLoggedIn, hasTicket, isAuthorized, userProfile, getUserProfile]);
```
Comment (smarcet) [inline src/routes/PrivateRoute.js:54]: get user profile is not a dependency its defined on other file and not at render state
please check this
https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies
"It is only safe to omit a function from the dependency list if nothing in it (or the functions called by it) references props, state, or values derived from them"

**Review by smarcet — CHANGES_REQUESTED** [review]
Please review comments 
also i do think that *.json should not be on the PR
regards

File: src/routes/PrivateRoute.js (hunk 1 above)
Comment (santipalenque) [reply src/routes/PrivateRoute.js:54]: is triggering a warning Seba, that's why I added it. The text you pasted says you can omit, not you should omit

**Review by smarcet — APPROVED** [review]
looks good to me

**Pull request merged**